nevergrad/optimization/base.py

```diff
@@ -164,6 +170,22 @@ def num_tell_not_asked(self) -> int:
         """
         return self._num_tell_not_asked
 
+    def set_constraints_management(self, max_constraints_trials: int, constraint_penalization: float, constraint_penalty_exponent: float,
+            penalize_cheap_violations: bool, memorize_constraint_failures: bool):
+        """We try max_constraints_trials random explorations for satisfying constraints.
+        The finally chosen point, if it does not satisfy constraints, is penalized as shown in the tell function.
+        """
```

Comment:
Please improve the docstring, in the standard format. And avoid `We`

Reply:
I'm also not sure I would make this method public for now, since it may be unstable,
The docstring should state it.

Reply:
Modified accordingly.